example/build.sbt

```diff
@@ -2,12 +2,12 @@ name := "example"
 
 version := "1.0"
 
-scalaVersion := "2.12.8"
```

Comment:
Currently it's fair to say that 2.12 is still our main version. Did you make 2.13 the main one here for any reason in particular?

Reply:
At the time I wishfully thought I could update the documentation to Scala 2.13, which would enable us to move other CI steps to Scala 2.13 and make 2.13 more "defaultish". I agree 2.12 is a better main version for now. I'll change it.

Reply:
Made 2.12 the main version of the example project in f38abfb.